lib/apimatic-core/api_call.rb

```diff
@@ -7,6 +7,9 @@ def new_builder
       ApiCall.new(@global_configuration, logger: @endpoint_logger.logger)
     end
 
+    # Initializes a new instance of ApiCall.
+    # @param [GlobalConfiguration] global_configuration An instance of GlobalConfiguration.
+    # @logger An optional logger to log execution of program.
```

Comment:
@logger should be replaced with @param decorator to allow IDE to parse the documentation properly.